about.html

```diff
@@ -0,0 +1,127 @@
+                </div>
+            </section><br><br><br>
+        </section>
+        <section class="program-about bg-white">
+            <div class="container">
+                <br>
+                <h4 class="text-center section-title-black">Music Festival<br>On Winter</h4>
+                <hr class="line-name">
+                <p>This was intended to help people from different ages,<br>religions, and cultures to get together and enjoy themselves</p><br>
+                <div class="container"><img src="assets/trebleClef.png" class="main-logo" width="90%" alt="logo"></div><br>
+            </div>
+        </section><hr>
+        <section class="past-program bg-white">
+            <div class="container">
+                <br>
+                <h4 class="text-center section-title-black">See the past Music Festivals</h4>
+                <hr class="line-name">
+                <p>Explore this in the name of fun and happiness, I dare you.</p>
+                <div class="past-summit">
+                    <div class="summit-card summit-one text-center mt-3">
+                        <div class="summit-sub">
+                            <h2 class="text-white">2016</h2>
+                            <p class="text-white"> Gauteng Concert</p>
+                        </div>
+                    </div>
+                    <div class="summit-card summit-two text-center mt-3">
+                        <div class="summit-sub">
+                            <h2 class="text-white">2016</h2>
+                            <p class="text-white"> Gauteng Concert</p>
+                        </div>
+                    </div>
+                </div>
+                <br><br>
+            </div>
+        </section>
+        <section class="more-partners more-partners-about bg-secondary">
+            <div class="container"><br>
+                <h4 class="text-center section-title-white">Partner</h4><hr class="line-name">
+                <div class="container p-div text-center">
+                    <div class="first-group">
+                        <h4 class="first-p">Spotify</h4>
+                        <h4 class="second-p">&nbsp YouTube</h4>
+                        <h4 class="third-p">&nbsp DStv</h4>
+                    </div>
+                    <div class="first-group">
+                        <h4 class="fourth-p">&nbsp Amazon</h4>
+                        <h4 class="fifth-p">&nbsp TikTok</h4>
+                    </div>
+                </div>
+            </div><br><br>
+        </section>
+    </main>
+    <footer class="footer">
+        <div class="container bottom-part">
+            <img src="assets/musicfestival.png" width="100px" alt="Music Festival">
+            <div class="pl-5 bottom-txt"><br>2023. Some Rights Reserved.<br>
+                <i class="last-text">For more info visit our site: https://www.musicfestival.com</i>
+            </div>
+        </div><br>
+    </footer>
```

Comment:
- [ ] This footer on dektop screen has background-color `#2e2e2e`. kindly consider changing color this footer on desktop screen.
your design
![l](https://github.com/Kgomotso196/FirstCapstoneProject/assets/96683843/4fc6af42-9ded-488b-b933-83a97628f9c2)

behance design
![p](https://github.com/Kgomotso196/FirstCapstoneProject/assets/96683843/ca0e033d-f700-4a76-bc21-3164449ea121)